build.cake

```diff
@@ -12,7 +12,11 @@ Task("Clean")
 {
 	if (DirectoryExists("./artifacts"))
 	{
-		DeleteDirectory("./artifacts", true);
+		DeleteDirectory("./artifacts", new DeleteDirectorySettings {
+		Recursive = true,
+		Force = true
+	});
```

Comment:
Some indentation on the last 3 lines here please. And remove the line trailing line after this.